src/main/java/com/moodybluez/enterprise/dto/Entry.java

```diff
@@ -4,7 +4,10 @@
 
 public @Data
 class Entry {
+    private int entryID;
     private int moodID;
-    private String reasonDesc;
-    public Date date;
+    private int weekDayID;
+    private String description;
+    private Date date;
```

Comment:
Updated to more generic naming Convention format.
Note - Here description is the actual description input from user, so it makes more sense.
Citation - "a name that tells the reader of the program what the variable represents, Keep the variable names short avoid including metadata" - R. Hamedy